Assets/UI/Panels/citypaneloverview.lua

```diff
@@ -546,50 +546,75 @@ function ViewPanelAmenities( data:table )
 
     g_kAmenitiesIM:ResetInstances();
 
-    --Luxuries
+    -- Luxuries
     CQUI_BuildAmenityBubbleInstance("ICON_IMPROVEMENT_BEACH_RESORT", data.AmenitiesFromLuxuries, "LOC_PEDIA_RESOURCES_PAGEGROUP_LUXURY_NAME");
-    --Civics
+    
+    -- Civics
     if GameCapabilities.HasCapability("CAPABILITY_CITY_HUD_AMENITIES_CIVICS") then
         CQUI_BuildAmenityBubbleInstance("ICON_NOTIFICATION_CONSIDER_GOVERNMENT_CHANGE", data.AmenitiesFromCivics, "LOC_CATEGORY_CIVICS_NAME");
     end
-    --Entertainment
+    
+    -- Entertainment
     CQUI_BuildAmenityBubbleInstance("ICON_PROJECT_CARNIVAL", data.AmenitiesFromEntertainment, "LOC_CQUI_CITY_ENTERTAINMENT");
-    --Great People
+    
+    -- Great People
     if GameCapabilities.HasCapability("CAPABILITY_CITY_HUD_AMENITIES_GREAT_PEOPLE") then
         CQUI_BuildAmenityBubbleInstance("ICON_NOTIFICATION_CLAIM_GREAT_PERSON", data.AmenitiesFromGreatPeople, "LOC_PEDIA_CONCEPTS_PAGEGROUP_GREATPEOPLE_NAME");
     end
-    --Relgion
+
+    -- City-States
+	if GameCapabilities.HasCapability("CAPABILITY_CITY_HUD_AMENITIES_CITY_STATES") then
+        if data.AmenitiesFromCityStates > 0 then
+            CQUI_BuildAmenityBubbleInstance("ICON_GREAT_PERSON_CLASS_WRITER", data.AmenitiesFromCityStates, "LOC_CITY_STATES_TITLE");
+        end
+	end
+    
+    -- Religion
     if GameCapabilities.HasCapability("CAPABILITY_CITY_HUD_AMENITIES_RELIGION") then
         CQUI_BuildAmenityBubbleInstance("ICON_UNITOPERATION_FOUND_RELIGION", data.AmenitiesFromReligion, "LOC_UI_RELIGION_TITLE");
     end
-    --National Parks
+    
+    -- National Parks
     if GameCapabilities.HasCapability("CAPABILITY_CITY_HUD_AMENITIES_NATIONAL_PARKS") then
         CQUI_BuildAmenityBubbleInstance("ICON_UNITOPERATION_DESIGNATE_PARK", data.AmenitiesFromNationalParks, "LOC_PEDIA_CONCEPTS_PAGE_TOURISM_4_CHAPTER_CONTENT_TITLE");
     end
-    --War Weariness
+    
+    -- Starting Era
+	if(data.AmenitiesFromStartingEra > 0) then 
+        CQUI_BuildAmenityBubbleInstance("ICON_GREAT_PERSON_CLASS_SCIENTIST", data.AmenitiesFromStartingEra, "LOC_GAME_START_ERA");
+	end
+    
+    -- Improvements
+    CQUI_BuildAmenityBubbleInstance("ICON_IMPROVEMENT_PASTURE", data.AmenitiesFromImprovements, "LOC_IMPROVEMENT_NAME");
```

Comment:
FWIW: I used ICON_CITYSTATE_INDUSTRIAL here, and added displaying this with the conditional if (data.AmenititesFromImprovements > 0), which mirrors the unmodified file

Reply:
LOC_PEDIA_IMPROVEMENTS_PAGEGROUP_IMPROVEMENTS_NAME is the pluralized version (_Improvements_ instead of _Improvement_)

Reply:
There is a section just below Amenities that shows sources of Housing. It so happens that some sources are the same e.g. improvements are one of them. Imho the icons and names should be consistent in both those sections. Housing is using singular form. We can choose whatever standard but we must be consistent.
In general I am NOT fond of using Civilopedia LOCs. Too unstructured, too volatile, to contextual. I'd rather use standard HUD LOCs that tend to be more strict.